common/src/main/java/com/microsoft/identity/common/internal/providers/microsoft/azureactivedirectory/AzureActiveDirectory.java

```diff
@@ -110,6 +110,14 @@ public static AzureActiveDirectoryCloud getAzureActiveDirectoryCloud(final URL a
         return sAadClouds.get(authorityUrl.getHost().toLowerCase(Locale.US));
     }
 
+    /**
+     * @param preferredCacheHostName String
+     * @return AzureActiveDirectoryCloud
+     */
+    public static AzureActiveDirectoryCloud getAzureActiveDirectoryCloud(final String preferredCacheHostName) {
```

Comment:
nit : rename `getAzureActiveDirectoryCloudFromHostName` to distinguish from the method above

Reply:
Addressed